lnrpc/routerrpc/router.proto

```diff
@@ -386,14 +386,22 @@ message PairHistory {
 }
 
 message PairData {
-    /// Time stamp of last result.
-    int64 timestamp = 1 [json_name = "timestamp"];
+    /// Time of last failure.
```

Comment:
could RPC changes have been made its own commit?

Reply:
difficult to do, because the internal data model changes and won't match the not-yet-updated rpc protos